demo/static/sass/demo-ltr.scss

```diff
@@ -9,6 +9,11 @@
 // ------------------------------
 $layout-direction: ltr;
 
+// ------------------------------
+// #LIB
+// ------------------------------
+@import '../../../node_modules/bi-app-sass/bi-app/bi-app-ltr';
```

Comment:
Thanks, @andy-armstrong! That's damn good feedback and simplifying this more makes sense.

I've gone ahead and done that with this ``fixup!`` commit (https://github.com/edx/ux-pattern-library/commit/a4dc34ddfc4d5b706391d9557af35d3d5feecd76). The README file compilation examples have also bee updated. How does this sound?